scripts/e2e/e2e-datadog-logs/compose.yaml

```
@@ -0,0 +1,100 @@
+version: '3'
+
+services:
+  # Generates random log data for consumption by the custom Agent check
+  log_generator:
+    image: docker.io/mingrammer/flog
+    command:
+      - "-f"
+      - "json"
+      - "-n"
+      - "1000"
+      - "-t"
+      - "log"
+      - "-o"
+      - "/var/log/a_custom.log"
+    volumes:
+      - log_path:/var/log/
+
+  # Receives log data from the `datadog-agent-vector` service and sends
+  # to the `fakeintake-vector` service.
+  vector:
+    depends_on:
+      - log_generator
+      - fakeintake-vector
+    build:
+      context: ${PWD}
+    # re-using the integration test runner image since it already has
+    # compiled vector on it.
+    image: ${CONFIG_VECTOR_IMAGE}
+    environment:
+      - FEATURES=e2e-tests-datadog
+    working_dir: /home/vector
+    network_mode: host
+    command:
+      - "/usr/bin/vector"
+      - "-vvv"
+      - "-c"
+      - "/home/vector/tests/data/e2e/datadog/logs/vector.toml"
+    volumes:
+      - ${PWD}:/home/vector
+
+  # Tails a custom log created by `log_generator` and sends log data to
+  # the `fakeintake-agent` service
+  datadog-agent:
+    image: docker.io/datadog/agent:${CONFIG_VERSION}
+    depends_on:
+      - log_generator
+      - fakeintake-agent
+    environment:
+    - DD_API_KEY=${TEST_DATADOG_API_KEY:?TEST_DATADOG_API_KEY required}
+    - DD_HOSTNAME=datadog-agent
+    - DD_ENABLE_PAYLOADS_EVENTS=false
+    - DD_ENABLE_PAYLOADS_SERVICE_CHECKS=false
+    - DD_CONTAINER_EXCLUDE="name:.*"
+    volumes:
+    # The Agent config file
+    - ../../../tests/data/e2e/datadog/logs/agent_only.yaml:/etc/datadog-agent/datadog.yaml
+    # The custom logs check
+    - ../../../tests/data/e2e/datadog/logs/logs.conf.d:/conf.d:ro
```

Using the `vector` block as an example, could we use `${PWD}` here to simplify things?

```suggestion
    - ${PWD}/tests/data/e2e/datadog/logs/agent_only.yaml:/etc/datadog-agent/datadog.yaml
    # The custom logs check
    - ${PWD}/tests/data/e2e/datadog/logs/logs.conf.d:/conf.d:ro
```